ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/BeaconState.java

```diff
@@ -29,42 +29,42 @@
 
 public class BeaconState {
   // Misc
-  private UnsignedLong slot;
-  private UnsignedLong genesis_time;
-  private Fork fork; // For versioning hard forks
+  protected UnsignedLong slot;
```

Comment:
If you aren't making a new class that inherits from BeaconState stores previousBlock, then we should probably just leave these member variables as is.